Dockerfile

```diff
@@ -44,6 +44,14 @@ ENV PATH="${PATH}:/tmp/linguist/bin"
 RUN DEBIAN_FRONTEND=noninteractive TZ=Etc/UTC apt-get install -y python3-packaging
 
 # cassandra
+RUN if [ -z "$RUN_BUILT_IN_CASSANDRA" ]; then \
```

Comment:
what does it fix? I don't see any changes on starting cassandra inside container